mmedit/datasets/registry.py

```diff
@@ -1,4 +1,12 @@
-from mmcv.utils import Registry
+import warnings
 
-DATASETS = Registry('dataset')
-PIPELINES = Registry('pipeline')
+from .builder import DATASETS, PIPELINES
+
+__all__ = ['DATASETS', 'PIPELINES']
+
+warnings.simplefilter('once', DeprecationWarning)
+warnings.warn(
+    'Registries (DATASETS, PIPELINES) have been moved to '
+    'mmedit.datasets.builder. Importing from '
+    'mmedit.models.registry will be deprecated in the future.',
```

Comment:
add pr link in the warning `details see https://github.com/open-mmlab/mmediting/pull/446`